src/components/header/header.jsx

```diff
@@ -1,16 +1,21 @@
+//@ts-check
 import React from 'react';
 import { ShoppingCartIcon, HeartIcon, SunIcon, HalfMoonIcon } from '../../assets/svg/svg';
 import { useTheme } from '../../context/theme/theme-context';
 import { Link, NavLink } from 'react-router-dom';
 import '../css/header.css';
 import { useWishlist } from '../../context/wishlist/wishlist-context';
 import { useMyCart } from '../../context/mycart/mycart-context';
+import { useModal } from '../../context/modal/modal-context';
+import { useAuth } from '../../context/auth/auth-context';
```

Comment:
Suggestion : To keep an index.js in context folder,
then these 4 lines will reduce to import {useWishlist, useMyCart, useModal, useAuth} from "../../context"

Reply:
I will be doing it while refactoring the code thanks for the suggestion